4_advanced_usage/77_distributed_rudder.txt

```diff
@@ -73,6 +73,111 @@ cd rudder-multiserver-setup
 
 ----
 
+==== Install rudder-db manually
+
+The setup is done in two places: on the database server and on the Rudder root server.
+This section allows installing a separate databse only without splitting the rest of the server components
+like when using the rudder-multiserver-setup script.
+It also allows moving an existing database to another server.
```

Comment:
I didn't see any explanations about that. If we don't document it, don't say it.